source/features/conversation-activity-filter.tsx

```diff
@@ -141,12 +141,13 @@ async function addWidget(state: State, anchor: HTMLElement): Promise<void> {
 		return;
 	}
 
+	// TODO: Use `<anchored-position>` instead
 	// Try to place the dropdown to the left https://github.com/refined-github/refined-github/issues/5450#issuecomment-1068284635
+	await delay(100); // Let `clean-conversation-headers` run first
```

Comment:
- Fixes regression of https://github.com/refined-github/refined-github/issues/5450

<img width="488" alt="Screenshot 11" src="https://github.com/user-attachments/assets/ef5f45b1-87c4-434d-9db7-9fd4b583c757">

After:

<img width="488" alt="Screenshot 12" src="https://github.com/user-attachments/assets/1dfa9090-5daa-484e-9761-2b69381f0dec">


cc @kidonng it would be great if you could apply https://github.com/refined-github/refined-github/pull/7496 to this feature too, using `anchored-position`

Reply:
cc @kidonng it would be great if you could apply #7496 to this feature too, using anchored-position